modules/lib/lib-schema/src/main/resources/lib/xp/examples/schema/updateStyles.js

```diff
@@ -0,0 +1,68 @@
+var schemaLib = require('/lib/xp/schema');
+var assert = require('/lib/xp/testing');
+
+/* global log*/
+
+let resource = '<?xml version="1.0" encoding="UTF-8"?>' +
+               '<styles css="assets/styles.css" xmlns="urn:enonic:xp:model:1.0">' +
+               '<style name="warning">' +
+               '<display-name i18n="warning.displayName">Warning</display-name>' +
+               '</style>' +
+               '<image name="editor-width-auto">' +
+               '<display-name i18n="editor-width-auto-text">Override ${width}</display-name>' +
+               '</image>' +
+               '<image name="editor-style-cinema">' +
+               '<display-name i18n="editor-style-cinema-text">Cinema</display-name>' +
+               '<aspect-ratio>21:9</aspect-ratio>' +
+               '<filter>pixelate(10)</filter>' +
+               '</image>' +
+               '</styles>'
+
+
+// BEGIN
+// Update virtual styles.
+var result = schemaLib.updateStyles({
+    application: 'myapp',
+    resource: resource
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=9736743)
```suggestion
    resource
```